espnet2/asr/espnet_model.py

```diff
@@ -182,7 +183,11 @@ def collect_feats(
         text: torch.Tensor,
         text_lengths: torch.Tensor,
     ) -> Dict[str, torch.Tensor]:
-        feats, feats_lengths = self._extract_feats(speech, speech_lengths)
+        if isinstance(self.normalize, GlobalMVN):
```

Comment:
Can someone check this part?
I modified this because it is pain to run `collect_feats` for HuBERT / Wav2Vec2 frontend. On the other hand, if `utt_mvn` is used, then I feel it is not necessary to run this step, although I'm not sure.